# all_the_seqs_and_etc

*testing*
```

A Work in Progress
These are notes for myself

->> Super Rough Draft <<-
(Organizing what I know, new stuff etc.)

** very very rough draft, trying to organize a lot of information **

1. Choose a sequencing/profiling method
2. Add branch to tree

Probe, Unknown Sample
├── DNA                                         
│    ├── single cell
│    ├── bulk
│	   │    ├── molecular resolution
│	   │    │   └── 
│    ├── blot
│    │   └── southern blot
│    ├── array
├── RNA
│    ├── molecular resolution
│    │   └── RNA-seq
│    ├── blot
│    │   └── northern blot
├── Protein
│    ├── molecular resolution
│    │   └── ChIP-seq
│    │   └── CLIP-seq
│    ├── blot
│    │   └── western blot

Profiling
├── Regulatory Activity
│    ├── molecular base resolution
│    │   └── ATAC-seq
│    ├── chemical marking
│    │   └── MeRIP-Seq
├── Spatial Transcriptomics
│    ├── molecular resolution
│    ├── visualization via fluorophores
│    │   └── 
├── RNA Structure Profile
│    ├── molecular resolution
│    │   └── 
├── Genotyping
├── Gene Expression Profiling

Analysis (Data Analysis / Methods)
├── Gene Enrichment
├── Interactome
│    ├── RNA-RNA
│    ├── RNA-Protein
├── 

Statistical Methods / Models
├── Gene Enrichment
├── Nanopore Basecalling



```


table ideas

name / input / output ? / special equipment / figures / best use cases 

advantages / limitations / application

data generated from sequencer per method in GB/TB/PB

method's avg disk space or memory used for analysis

method's recommended aligner or workflow and etc.

some of my own drawings

* ~ a commit a day, in the evening

[XYZeq](https://advances.sciencemag.org/content/7/17/eabg4755?intcmp=trendmd-adv)

[DART-seq](https://www.researchsquare.com/article/nprot-7063/v1)
* Droplet Assisted RNA Targeting by single cell sequencing

original RNA seq paper

original ChIP seq paper

etc
